experiments/compositions/samples/AppTeam/composition/appteam.yaml

```diff
@@ -237,8 +235,17 @@ spec:
             properties:
               project:
                 type: string
+              adminUser:
+                type: string
+              billingUser:
+                type: string
+              folder:
+                type: string
             required:
             - project
+            - adminUser
+            - billingAccount
```

Comment:
is it possible to make the billingAccount optional? So that in the automated e2e test, I do not have to specify a billingAccount in the facadeCR

Reply:
actually it is okay, I can alter the CRD a little bit in the test

Reply:
For this composition (which includes the bucket) we need a billing.